experiments/synthea-breast-cancer/winter2022/Differentially-Private/quickstart.sh

```diff
@@ -250,6 +259,35 @@ if [[ ${TO_INGEST} -eq 1 ]]; then
     sleep ${SLEEP_TIME}
 fi
 
+# Add Experiment to Containers
+echo
+docker-compose up -d
+
+## Kill All Running FL-* containers
```

Comment:
The user should be informed that running `fl-*` containers are going to be killed. Add a note about this to the `help` for this script, and also have the script `echo` the fact that it is about to perform the kill.